test/e2e/metallb_test.go

```diff
@@ -0,0 +1,279 @@
+//go:build e2e
+// +build e2e
+
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/aws/eks-anywhere/pkg/curatedpackages"
+	"github.com/aws/eks-anywhere/test/framework"
+	"github.com/stretchr/testify/suite"
+)
+
+type Suite struct {
+	suite.Suite
+	cluster *framework.ClusterE2ETest
+}
+
+func TestPackagesMetalLB(t *testing.T) {
+	suite.Run(t, new(Suite))
+}
+
+func (suite *Suite) SetupSuite() {
+	t := suite.T()
+	suite.cluster = framework.NewClusterE2ETest(t, framework.NewDocker(t),
+		framework.WithPackageConfig(t, EksaPackageBundleURI, EksaPackageControllerHelmChartName,
+			EksaPackageControllerHelmURI, EksaPackageControllerHelmVersion,
+			EksaPackageControllerHelmValues),
+	)
+}
+
+func (suite *Suite) TestPackagesMetalLB() {
+	// This should be split into multiple tests with a cluster setup in `SetupSuite`.
+	// This however requires the creation of utilites managing cluster creation.
+	t := suite.T()
+	suite.cluster.WithCluster(func(test *framework.ClusterE2ETest) {
+		test.InstallCuratedPackagesController()
```

Comment:
OK, it won't because you've changed the behavior of InstallCuratedPackagesController, which makes this really more of a MaybeInstallCuratedPackagesController.

I'd prefer to see:

```go
if !test.IsCuratedPackagesControllerInstalled(){
    test.InstallCuratedPackagesController()
}
```